### PR TITLE
Add baseUri path also when creating  _bulk uri

### PR DESCRIPTION
**Pull request by pramodShehan5**
References
- #3019

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @pramodShehan5,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

**Comment by pramodShehan5** [comment]
This is the exact issue.
https://github.com/akka/alpakka/issues/3019

**Comment by ennru** [comment]
CLA was signed.

File: elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala [hunk 1]
```diff
@@ -79,7 +79,7 @@ private[elasticsearch] final class ElasticsearchSimpleFlowStage[T, C](
       log.debug("Posting data to Elasticsearch: {}", json)
 
       if (json.nonEmpty) {
-        val uri = baseUri.withPath(Path(endpoint))
+        val uri = baseUri.withPath(Path(baseUri.path.toString() + endpoint))
```
Comment (ennru) [inline elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala:82]: This doesn't look right, `baseUri` should already be there as this attaches to it.

Comment (pramodShehan) [reply elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala:82]: if `baseUri` is `https://test.com:9200/elasticsearch` , it is ignoring` /elasticsearch` path.

Comment (pramodShehan5) [reply elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala:82]: if `baseUri` is` https://test.com:9200/elasticsearch`, it is ignoring` /elasticsearch` path

Comment (pramodShehan5) [reply elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala:82]: image

Comment (ennru) [reply elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala:82]: I believe what you are looking for is `baseUri.withPath(baseUri.path / endpoint)` but can't verify that right now.

**Comment by ennru** [comment]
Superseded by
- #3042

**Pull request closed without merging**